src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.ts

```diff
@@ -83,10 +174,12 @@ export class CopyQuestionsFromOtherSessionsModalComponent implements OnInit {
   }
 
   /**
-   * Checks whether there are any selected question.
+   * Checks whether there are any selected questions.
    */
   get hasAnyQuestionsToCopySelected(): boolean {
-    return this.questionToCopyCandidates.find((c: QuestionToCopyCandidate) => c.isSelected) !== undefined;
+    return this.feedbackSessionTabModels.reduce((a: boolean, b: FeedbackSessionTabModel) => {
+      return a || b.questionsTableRowModels.find((c: QuestionToCopyCandidate) => c.isSelected) !== undefined;
```

Comment:
I think `b.questionsTableRowModels.find((c: QuestionToCopyCandidate) => c.isSelected) !== undefined` can be replaced by `!!b.questionsTableRowModels.find((c: QuestionToCopyCandidate) => c.isSelected)`